pkg/agent/agent.go

```diff
@@ -183,7 +187,7 @@ func (i *Initializer) setupOVSBridge() error {
 		return err
 	}
 
-	if err := i.validateSupportedDPFeatures(); err != nil {
+	if err := i.validateSupportedDPFeatures(ovsctl.NewClient(i.ovsBridge)); err != nil {
```

Comment:
`Initializer` has a member variable `ovsCtlClient`, we should use it to call `GetDPFeatures`. And it can be mocked when creating initializer for test.